internal/net/call/msg.go

```diff
@@ -60,10 +60,13 @@ const currentVersion = initialVersion
 //    version  [4]byte
 //
 // requestMessage:
-//    headerKey    [16]byte   -- fingerprint of method name
-//    deadline      [8]byte   -- zero, or deadline in microseconds
-//    traceContext [25]byte   -- zero, or trace context
-//    remainder               -- call argument serialization
+//    headerLen         [4]byte       -- length of the encoded header
+//    header            [length]byte  -- encoded header information
+//      headerKey       [16]byte      -- fingerprint of method name
```

Comment:
The following part looks wrong? I think you need follow header by remainder and then extend the comment after that to say something like:

The header is encoded using Service Weaver's encoding format for a type that looks like:

```
struct {
  ...fill this in...
}
```

Reply:
Good idea. Done